Media queries module (#31903)
mdn/content: files/en-us/web/css/css_media_queries/using_media_queries_for_accessibility/index.md

Blinking and flashing animation can be problematic for people with cognitive concerns such as Attention Deficit Hyperactivity Disorder (ADHD). Additionally, certain kinds of motion can be a trigger for Vestibular disorders, epilepsy, and migraine and Scotopic sensitivity. The [`prefers-reduced-motion`](/en-US/docs/Web/CSS/@media/prefers-reduced-motion) media query enables providing an experience with fewer animations and transitions to users who have set their operating system's accessibility preferences to reduce motion.

> Reducing animations or switching animation off completely based on the user's preference can also benefit users with low battery or low-end phones or computers.

Reviewer: ```suggestion
Reducing animations or switching animation off completely based on the user's preference can also benefit users with low battery or low-end devices.
```

}
```

> The value of `prefers-reduced-motion` is `reduce`, not "none". This preference does not mean all animations must be removed, such as could be set with `* {animation: none !important;}`. Rather, users expect motion animation, including those triggered by user interaction, to be disabled unless the animation is essential to the functionality or the information being conveyed (see [WCAG: Animation from Interactions](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)).

Reviewer: ```suggestion
The value of `prefers-reduced-motion` is `reduce`, not "none". This preference does not mean all animations must be removed, which could be achieved with `* {animation: none !important;}`. Rather, users expect motion animation, including those triggered by user interaction, to be disabled unless the animation is essential to the functionality or the information being conveyed (see [WCAG: Animation from Interactions](https://www.w3.org/WAI/WCAG21/Understanding/animation-from-interactions.html)).
```
